Formula/ecl.rb

```diff
@@ -14,19 +14,39 @@ class Ecl < Formula
   end
 
   depends_on "gmp"
+  depends_on "bdw-gc"
 
   def install
-    system "./configure", "--prefix=#{prefix}",
+    # These environment variables are necessary or gmp / boehmgc won't be found
+    # No idea why
+    gmp = Formula["gmp"]
+    boehmgc = Formula["boehmgc"]
+    ENV.append "CFLAGS", "-I#{gmp.opt_include} -I#{boehmgc.opt_include}"
+    ENV.append "CXXFLAGS", "-I#{gmp.opt_include} -I#{boehmgc.opt_include}"
+    ENV.append "LDFLAGS", "-L#{gmp.opt_lib} -L#{boehmgc.opt_lib}"
+
+    # https://gitlab.com/embeddable-common-lisp/ecl/tree/revert-make-split
+    ENV.deparallelize
+
+    system "./configure",
+                          "--prefix=#{prefix}",
                           "--enable-threads=yes",
-                          "--with-system-gmp=yes"
+                          "--with-dffi=included",
+                          "--with-cxx",
+                          "--enable-gmp=system"
     system "make"
     system "make", "install"
+    system "make", "check"
```

Comment:
```
  --with-gmp-prefix=path  prefix for system GMP includes and libraries
  --with-gmp-incdir=path  path to system GMP includes (overrides prefix)
  --with-gmp-libdir=path  path to system GMP libraries (overrides prefix)
```
these are flags to set the prefix. I've deprecated some flags to make flags consistent, so you may now do `--enable-gmp=system` (previously we had `--with-system-gmp`, `--with-C-gmp`, and `--enable-gmp=included`)